### PR TITLE
Automatic prefill user mobile phone number

### LINKED ISSUE #152: Automatic prefill mobile phone number
# Description
as user I'd like the mobile phone number to be automatically prefilled when register and login. investigate a way to get device phone number and prefill them.

<br />

# Type
- [x] feature
- [ ] bug

<br />

### PR DESCRIPTION
**Pull request by wndisra**
Fixes #152 

### Breakdown:
- [x] Add read phone state permission to read SIM data
- [x] Add react-native-sim-data package
- [x] Add auto prefill phone number feature in register and login

### Note:
There is no guarantee that this package will get the user phone number precisely.

### CODE REVIEW

**Comment by wndisra** [comment]
Done. Please review it @andy-shi88

File: app/containers/Main/index.js [hunk 1]
```diff
@@ -71,11 +72,14 @@ class Main extends Component {
...
+    const countryCode = RNSimData.getCountryCode();
+
     AccountKit.configure({
-      countryWhitelist: [ 'ID' ],
-      defaultCountry: 'ID',
+      countryWhitelist: ['ID'],
+      defaultCountry: countryCode.toUpperCase() || 'ID',
       initialPhoneCountryPrefix: '+62',
-      initialPhoneNumber: '87809000750'
+      initialPhoneNumber: phoneNumber || '87809000750'
```
Comment (andy-shi88) [inline app/containers/Main/index.js]: hm, can you add the default value to empty string instead of the above random number?

**Review by andy-shi88 — CHANGES_REQUESTED** [review]

File: app/containers/Main/index.js (hunk 1 above)
Comment (wndisra) [reply app/containers/Main/index.js]: Sure. Give me a minute

**Comment by andy-shi88** [comment]
@wlisrausr  seems like there are some problem related the dependencies, let us check cc: @caudaganesh

**Comment by wndisra** [comment]
Okay. Let me know If I have to add or modify something @andy-shi88

**Comment by andy-shi88** [comment]
let's put this on hold for now.

**Pull request closed without merging**